config/application.rb

```diff
@@ -25,6 +25,14 @@ class Application < Rails::Application
     # Only loads a smaller set of middleware suitable for API only apps.
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
+
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins '*'
+        resource '*', :headers => :any, :methods => [:get, :post, :options]
+      end
```

Comment:
Are we going to configure this to accept only certain origin paths later or allow all origins and come up with another way of securing the app?